src/Console/Commands/MakeFieldtype.php

```diff
@@ -89,6 +99,62 @@ protected function buildVueComponent($name)
         return $component;
     }
 
+    /**
+     * Wire up addon JS.
+     *
+     * @param string $addon
+     */
+    protected function wireUpAddonJs($addon)
+    {
+        $addonPath = $this->getAddonPath($addon);
+
+        $files = [
+            'addon/webpack.mix.js.stub' => 'webpack.mix.js',
+            'addon/package.json.stub' => 'package.json',
+            'addon/addon.js.stub' => 'resources/js/addon.js',
+        ];
+
+        $data = [
+            'name' => $this->getNameInput(),
+            'package' => $this->package,
+            'root_namespace' => $this->rootNamespace(),
+        ];
+
+        $this->info('Scaffolding the Vue component boilerplate...');
+        $this->info('--------------------------------------------------------------');
+        foreach ($files as $stub => $file) {
+            $path = $addonPath.'/'.$file;
+            $this->createFromStub($stub, $path, $data);
+            $this->line($path);
```

Comment:
If you've already created a fieldtype, there would already be a webpack.mix.js, package.json, and addon.js files.

The output though sorta suggests you just created them, even though nothing happens. It just skips over files that already exist.

![image](https://user-images.githubusercontent.com/105211/125976665-3e041ae9-52cd-4b0a-b50b-7edf5b8d17a4.png)

Do we need to output each file? What if we made it more generic?

![image](https://user-images.githubusercontent.com/105211/125976996-be2d008c-7fa3-4d40-8e75-fd48bd16da00.png)

Also, in `addon.js`, the user is gonna need to add their second fieldtype.
In `webpack.mix.js`, they're gonna have to add their View component.
It's cool if it doesn't automatically do that, but there's no output telling them to.

Reply:
Sure, no biggie to make it more generic. 

Will tweak the other message to include the js files 👍

Reply:
As for `addon.js` – you can't add more components to an existing addon though, the flow doesn't support that.